code/game/machinery/doors/blast_door.dm

```diff
@@ -103,6 +105,14 @@
 	src.set_opacity(1)
 	sleep(15)
 	src.operating = 0
+	if(istype(src, /obj/machinery/door/blast/regular)) //regular blast doors are heavier and do more damage
+		for(var/mob/living/M in src.loc)
+			M.take_organ_damage(20)
```

Comment:
Please, do not do this. This will cause a LOT of issues, and assured death of anyone unlucky enough.

Reply:
you're mistaken here, take organ damage is a generic proc for damaging limbs. However, it is intended only for use when targeting a specific limb, and isn't the right thing here.

use take_overall_damage or apply_damage